core/src/main/java/hudson/PluginManager.java

```diff
@@ -425,6 +430,190 @@ public void run(Reactor reactor) throws Exception {
         }});
     }
 
+    protected @Nonnull Set<String> loadPluginsFromWar(@Nonnull String fromPath) {
+        return loadPluginsFromWar(fromPath, null);
+    }
+
+    protected @Nonnull Set<String> loadPluginsFromWar(@Nonnull String fromPath, @CheckForNull FilenameFilter filter) {
+        Set<String> names = new HashSet();
+
+        ServletContext context = Jenkins.getInstance().servletContext;
```

Comment:
findbugs will tell you there is a NPE here... as `Jenkins.getInstace()` is `checkForNull`